elasticsearch/templates/statefulset.yaml

```diff
@@ -109,14 +109,14 @@ spec:
         image: "{{ .Values.image }}:{{ .Values.imageTag }}"
         imagePullPolicy: "{{ .Values.imagePullPolicy }}"
         readinessProbe:
-{{ toYaml .Values.readinessProbe | indent 10 }}
+{{ toYaml .Values.readinessProbe.config | indent 10 }}
```

Comment:
Did you mean to leave this change in? Reading through #60 makes me believe this was leftover from testing a different approach. In any case this will fail with the current default values files and tests.